index.html

```diff
@@ -1528,28 +1540,103 @@
 
 
     <section id="h_numbers">
-      <h3><a href="#h_numbers">Arabic numbering</a>
+      <h3><a href="#h_numbers">Numbers</a>
       </h3>
 
-        <section id="h_Arabic_number_writing">
-          <h4><a href="#h_Arabic_number_writing">Arabic number writing</a> </h4>
-          <p>There are mostly two families of <span style="font-family: Waseem;">numerals</span>
-            in Arabic script. One known as European digits aka as Arabic Digits
-            (Unicode range U+0030-U+0039), the second is Arabic-Indic Digits
-            (Unicode range U+0660-U+0669). The latter further gave another
-            sub-notation for Persian/Urdu digits, the Extended (or Eastern)
-            Arabic-Indic Digits (Unicode range U+06F0-U+06F9), in which digits
-            4, 5 and 6 have another glyph. The following table summarizes those
-            families:</p>
-          <figure id="order_in_memory"> <img style="width: 465px; height: 95px;"
-              src="images/Arabic-numeral.png"
-              alt="Arabic Numerals">
-            <figcaption> Different notations for Arabic script digits </figcaption>
-          </figure>
-          <p>Digits on the first row are predominant in Western Arabic regions,
-            while the second row digits are used in most of the Middle-East
-            countries, sometime along with the former. Persian and Urdu mostly
-            use the third category.</p>
+      <section id="h_naming_conventions">
+	<h4><a href="#h_naming_conventions">Preferred Terminology</a></h4>
+	<p>Before entering this section in  we need to introduce few preferred terminological
+	  conventions for disambiguation and simplicity.</p>
+
+	<ul>
+	  <li><strong><span class="qterm">Arabic Numerals</span></strong> are
```

Comment:
Ah, rereading the paragraph, it seems that the plan is still to refer to them as European digits.  I think the latter name should be the bold one at the start of the paragraph, and 'Arabic numerals' should be used per "They are also referred to as 'Arabic Numerals' or 'Western Arabic Numerals'."